test/integration/features/basic.feature

```diff
@@ -64,6 +64,13 @@ Feature: Basic
       export "BAZ=BAT"
       """
 
+  Scenario: User is able to get url on running service command
+    Given Minishift has state "Running"
+     When executing "oc new-app https://github.com/openshift/nodejs-ex"
+      And executing "oc expose svc/nodejs-ex"
+      And executing "minishift openshift service nodejs-ex -n myproject --url"
+     Then stdout should match /nodejs-ex-myproject.(.*).nip.io/
```

Comment:
Does this test pass for you? I get

    ...................................F--...... 44
   
    --- Failed steps:
    
        Then  stdout should match /nodejs-ex-myproject.(.*).nip.io/ # features/    basic.feature:72
          Error: Actual output  doesn't contain match nodejs-ex-myproject.(.*).nip.io
    
    9 scenarios (8 passed, 1 failed)
    44 steps (41 passed, 1 failed, 2 skipped)